tests/baselines/reference/user/TypeScript-React-Native-Starter.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 1
+Standard output:
+node_modules/@types/react-native/index.d.ts(3895,26): error TS2693: 'setImmediate' only refers to a type, but is being used as a value here.
+node_modules/@types/react-native/index.d.ts(3896,28): error TS2693: 'clearImmediate' only refers to a type, but is being used as a value here.
```

Comment:
I still don't understand what is going on here.

Reply:
It seems like this is a good change since it turns out (thanks @weswigham!) that setImmediate/clearImmediate are IE/Edge/Node-only, and `@types/node` already includes them. @andy-ms, since you are doing a DefinitelyTyped sweep, do you know whether there are other packages besides react-native that need to be fixed?